src/Pages/Dashboard/_Dashboard.scss

```diff
@@ -0,0 +1,77 @@
+@import "../../Shared/Variables";
+
+.dashboard {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  // height: -webkit-calc(100% - 100px);  
+  
+  top: 50px;
+  width: 100%;
+  overflow: auto;
+
+  h1{
+    @include pageHeaderText($button-font-colour);
```

Comment:
cool that you're using scss and variables.

Reply:
Nicolai's handiwork :)
We found out later that we really shouldn't have bothered, and just used component styling >.<